components/dropzone/index.css

```diff
@@ -114,7 +114,7 @@ governing permissions and limitations under the License.
   }
 
   &.is-dragged {
-    --mod-drop-zone-border-style: var(--mod-drop-zone-border-style--dragged, solid);
+    --mod-drop-zone-border-style: var(--mod-drop-zone-border-style-dragged, solid);
```

Comment:
Similar to the note that @castastrophe left below for TextField, https://github.com/adobe/spectrum-css/pull/2132#discussion_r1329047344, would you mind adding the double-dashed var to the stack so that we don't have to do a breaking change for this component?